service/src/main/kotlin/app/cash/backfila/dashboard/CreateBackfillAction.kt

```diff
@@ -25,12 +26,33 @@ class CreateBackfillAction @Inject constructor(
   @ResponseContentType(MediaTypes.APPLICATION_JSON)
   // TODO allow any user
   @Authenticated(capabilities = ["users"])
+  fun createDefault(
```

Comment:
On second thought I think I should just remove these as these will be hit by the javascript and therefore will never be used. @shellderp happy to discuss more if there is a case I am forgetting but I think I should remove these.